jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -21,9 +21,11 @@ def buildExclusions = [
   // Linux -------------------------------------------------
   [ /^centos6/,                       releaseType, lt(8)   ],
   [ /^centos6/,                       anyType,     gte(12) ],
-  [ /centos[67]-(arm)?(64|32)-gcc48/, anyType,     gte(10) ],
-  [ /centos[67]-(arm)?(64|32)-gcc6/,  anyType,     lt(10)  ],
-  [ /centos6-32-gcc6/,                releaseType, gte(10) ], // 32-bit linux for <10 only
+  [ /^centos[67]-(arm)?(64|32)-gcc48/,anyType,     gte(10) ],
+  [ /^centos[67]-(arm)?(64|32)-gcc6/, anyType,     lt(10)  ],
```

Comment:
Just added the `^` for consistency.

Reply:
ok, but _are you sure_ there are no labels used anywhere with some cruft in front? I don't think there but we've been caught with overly strict regexes in the past (in select-compler.sh trying to identify all the benchmark machines iirc was the latest).